test/evp_kdf_test.c

```diff
@@ -616,7 +616,7 @@ static int test_kdf_pbkdf2_small_iterations_pkcs5(void)
 static int test_kdf_pbkdf2_invalid_digest(void)
 {
     int ret = 0;
-    EVP_KDF_CTX *kctx;
+    EVP_KDF_CTX *kctx = NULL;
```

Comment:
The same issue in in the Scrypt code below..

Reply:
Scrypt test should be OK as it doesn't allocate params